CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing
+
+We would absolutely love any pull requests to contribute materials to our RSS feed of curated materials about reproducibility. If you add resources to our feed, the contributions will be tweeted out via the [@ReproFeed](https://twitter.com/ReproFeed) twitter bot, and added to the directory on [reproduciblescience.org](https://reproduciblescience.org/directory).
+
+## Best Practices for Contributing
+* Make sure that the resource you are suggesting isn't already in the feed. 
+* Make sure the resource you are suggesting is open access. This means that anyone should be able to access, read, and/or download it without having to pay money or login.
+* Be sure to add at least one tag to the resource. You can find a list of tags in the README file in this repository.
+
+## Workflow
+The following was adapted from [ProjectPorcupine's](https://github.com/TeamPorcupine/ProjectPorcupine)'s [CONTRIBUTING.md](https://github.com/TeamPorcupine/ProjectPorcupine/blob/master/CONTRIBUTING.md). 
```

Comment:
I think it's fine to have people edit from the web interface, i.e. just point them to https://github.com/ViDA-NYU/reproducibility-news/edit/master/news.yaml

![screenshot_20180621_174226](https://user-images.githubusercontent.com/426784/41747015-7d7aa0e2-757a-11e8-96ed-c096d3fd0148.png)

Reply:
Ok, I can add that then. Hold on.

Reply:
I mean instead of all the Git steps :wink: